c/driver/postgresql/postgres_copy_reader.h

```diff
@@ -1181,14 +1181,14 @@ static inline ArrowErrorCode MakeCopyFieldWriter(const enum ArrowType arrow_type
 
 class PostgresCopyStreamWriter {
  public:
-  ~PostgresCopyStreamWriter() { ArrowArrayViewReset(array_view_.get()); }
+  ~PostgresCopyStreamWriter() { ArrowArrayViewReset(&array_view_.value); }
```

Comment:
hmm, should probably make the handle implement `get()` and `reset()` at some point so it feels more like a smart pointer